skinfer/schema_inferer.py

```diff
@@ -17,6 +18,8 @@ def load_samples_from_jsonlines(file_list):
     for filename in file_list:
         with gzopen(filename) as f:
             for line in f:
+                if isinstance(line, six.binary_type):
```

Comment:
so, I was using GzipFile which doesn't seem to support opening in text mode.
I just switched to `gzip.open(filename, 'rt')` and it works -- thanks for the hint! :)